### PR TITLE
Fix: title in GitHub Repos which already has been translated

### LINKED ISSUE #134: [Suggestion] Fix title in GitHub Repos
### Request
- There is a title **that already has been translated** in `GitHub Repos` but does **not refelct (영문) delection** in title.
- Target: `Guideline Contributions`, `Repository structure`, `Repository branching`

### PR DESCRIPTION
**Pull request by FacerAin**
### Request
There is a title that already has been translated in GitHub Repos but does not refelct (영문) delection in title.
Target: Guideline Contributions, Repository structure, Repository branching

### Related Issue
close #134

### CODE REVIEW

File: _data/sidebars/general_sidebar.yml [hunk 1]
```diff
@@ -130,11 +130,11 @@ entries:
         url: /ios_implementation.html
   - title: GitHub 저장소
     folderitems:
-      - title: Guidelines Contributions (영문)
```
Comment (ianychoi) [inline _data/sidebars/general_sidebar.yml:133]: Please translate into Korean

File: _data/sidebars/general_sidebar.yml [hunk 2]
```diff
@@ -130,11 +130,11 @@ entries:
...
+      - title: Guidelines Contributions
         url: /policies_adoption.html
-      - title: Repository structure (영문)
+      - title: Repository structure
```
Comment (ianychoi) [inline _data/sidebars/general_sidebar.yml:135]: Please translate into Korean

File: _data/sidebars/general_sidebar.yml [hunk 3]
```diff
@@ -130,11 +130,11 @@ entries:
...
         url: /policies_repostructure.html
-      - title: Repository branching (영문)
+      - title: Repository branching
```
Comment (ianychoi) [inline _data/sidebars/general_sidebar.yml:137]: Please translate the title into Korean

**Review by ianychoi — CHANGES_REQUESTED** [review]
Would you also translate the titles in Korean language? Thanks

**Comment by FacerAin** [comment]
I also finished translating the title into korean you said. Thank you :)

**Comment by ianychoi** [comment]
Looks good thank you

**Pull request merged**